packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -92,7 +93,7 @@ const BpkBottomSheet = ({
   return <BpkBreakpoint query={BREAKPOINTS.ABOVE_MOBILE}>
     {(isAboveMobile: boolean) =>
       <BpkDialogWrapper
-        ariaLabelledby={ariaLabelledby}
+        {...props}
```

Comment:
Pass down the a11y props (we don't know which one we'll have but TS assures we'll have one)